CUDADataFormats/TrackingRecHit/BuildFile.xml

```diff
@@ -3,6 +3,7 @@
 <use name="CUDADataFormats/Common"/>
 <use name="CUDADataFormats/SiPixelCluster"/>
 <use name="DataFormats/Common"/>
+<use name="GeometryCommonTopologies"/>
```

Comment:
I guess it should be
```suggestion
<use name="Geometry/CommonTopologies"/>
```

But do we need an entry also for header-only dependencies ?